lib/repositories/tracking/tracking_repository.dart

```diff
@@ -1,12 +1,7 @@
-import 'package:ax_dapp/analytics/event.dart';
+import 'package:ax_dapp/repositories/tracking/event.dart';
```

Comment:
As discussed, this should be in the `packages/tracking_repository`, `packages` being sibling with `lib`.